pkg/agent/flags.go

```diff
@@ -0,0 +1,13 @@
+package agent
+
+type flagInt interface {
+	DebugServerData() bool
+}
+
+var Flags flagInt = &initialFlagTest{}
+
+type initialFlagTest struct{}
+
+func (initialFlagTest) DebugServerData() bool {
+	return true
```

Comment:
This should be `false` or conditionalized before merge. But `true` is handy for debugging.